### PR TITLE
add param to set threshold for duration of last segment

### PR DESCRIPTION
**Pull request by terranvigil**

### CODE REVIEW

File: vod/segmenter.c [hunk 1]
```diff
@@ -117,9 +117,13 @@ segmenter_init_config(segmenter_conf_t* conf, vod_pool_t* pool)
...
 
 uint32_t
-segmenter_get_segment_count_last_short(segmenter_conf_t* conf, uint64_t duration_millis)
+segmenter_get_segment_count_last_short(
+	segmenter_conf_t* conf,
+	request_context_t* request_context,
+	uint64_t duration_millis)
 {
 	uint32_t result;
+	uint32_t mod;
```
Comment (jamescyeh) [inline vod/segmenter.c:126]: oh C, never change.

**Review by jamescyeh — APPROVED** [review]
lgtm!

**Review by krystalmejia24 — COMMENTED** [review]
Been a while since I've looked at some c!  The logic looks good to me. we're decrementing the segment count by 1 when appending so can't imagine what else we'd need to do here.

**Review by zsiec — APPROVED** [review]
The changes make sense to me, e2e tests should hopefully help us tell if we've missed anything here

**Pull request merged**